### ISSUE TITLE
playground-v2.5-1024px-aesthetic does not support StableDiffusionXLInpaintPipeline

### ISSUE THREAD

**Issue by yanchaoguo** [huggingface/diffusers#7484]
The generation effect is not good

vae = AutoencoderKL.from_pretrained("madebyollin/sdxl-vae-fp16-fix", torch_dtype=torch.float16)
pipe = StableDiffusionXLInpaintPipeline.from_pretrained(
    "playgroundai/playground-v2.5-1024px-aesthetic",
    # image_encoder=image_encoder,
    # feature_extractor=feature_extractor,
    torch_dtype=torch.float16, variant="fp16" , vae=vae
).to("cuda")

**Comment by sayakpaul** [comment]
Yeah because there's no inpainting checkpoint in the first place.
